### PR TITLE
Update recommended KOTS version to 1.65.0

### PR DESCRIPTION
**Pull request by soulchips**
# Description
Update recommended KOTS version to 1.65.0

# Reasons
https://circleci.atlassian.net/browse/SERVER-1696

### CODE REVIEW

**Review by thomasr8 — APPROVED** [review]

File: jekyll/_cci2/server-3-install-prerequisites.adoc [hunk 1]
```diff
@@ -45,7 +45,7 @@ Download and install the following software before continuing:
 | {helmversion} or greater
 | Kubernetes Package Management
 
-| KOTS: https://github.com/replicatedhq/kots/releases/download/v1.47.3/kots_darwin_amd64.tar.gz[Mac] or https://github.com/replicatedhq/kots/releases/download/v1.47.3/kots_linux_amd64.tar.gz[Linux].
+| KOTS: https://github.com/replicatedhq/kots/releases/download/v1.65.0/kots_darwin_amd64.tar.gz[Mac] or https://github.com/replicatedhq/kots/releases/download/v1.65.0/kots_linux_amd64.tar.gz[Linux].
 | {kotsversion} *
```
Comment (soulchips) [inline jekyll/_cci2/server-3-install-prerequisites.adoc:49]: where is this value stored? @circleci/docs-disco I wasn't sure how to update it

Comment (soulchips) [reply jekyll/_cci2/server-3-install-prerequisites.adoc:49]: @thomasr8 i'm hoping you might know

Comment (rosieyohannan) [reply jekyll/_cci2/server-3-install-prerequisites.adoc:49]: It's here: https://github.com/circleci/circleci-docs/blob/c2838d9ab54db77ce21c4ab81f7d7224e7713d43/jekyll/_config.yml#L81 and in the section below too (two places for PDF generation - I really need to fix this though!)

File: jekyll/_config.yml [hunk 2]
```diff
@@ -78,7 +78,7 @@ asciidoc:
       terraformversion: 0.15.4
       kubectlversion: 1.19
       helmversion: 3.4.0
-      kotsversion: 1.47.3
+      kotsversion: 1.65.3
```
Comment (lloydChris) [inline jekyll/_config.yml:81]: There is no x.y.3 out.
```suggestion
      kotsversion: 1.65.0
```

**Review by lloydChris — CHANGES_REQUESTED** [review]

File: jekyll/_config.yml (hunk 2 above)
Comment (soulchips) [reply jekyll/_config.yml]: ah thank you! accident

File: jekyll/_config.yml [hunk 3]
```diff
@@ -91,7 +91,7 @@ asciidoctor:
     terraformversion: 0.15.4
     kubectlversion: 1.19
     helmversion: 3.4.0
-    kotsversion: 1.47.3
+    kotsversion: 1.65.3
```
Comment (lloydChris) [inline jekyll/_config.yml:94]: This one too 😄 
```suggestion
    kotsversion: 1.65.0
```

**Review by lloydChris — CHANGES_REQUESTED** [review]

**Review by ryanwohara — APPROVED** [review]

**Review by atulsingh0 — APPROVED** [review]

**Pull request merged**